front-end/cypress/support/commands.ts

```diff
@@ -204,6 +206,31 @@ export function calendarSetValue(calendar: string, dateObj: Date = new Date()) {
   cy.shortWait();
 }
 
+export function runLighthouse(directory: string, filename: string) {
+  cy.lighthouse(
+    {
+      performance: 0,
+      accessibility: 90,
```

Comment:
Might be worth a discussion what we want to set this number to? Maybe we could also discuss if we want to set/track any of the other thresholds.